File: source/code/nodejs/account_management/create_account_mcf_hash.js
var picard = {
  username: 'jlpicard',
  email: 'capt@enterprise.com',
  givenName: 'Jean-Luc',
  surname: 'Picard',
  password: '$stormpath2$SHA-256$256$salt$hash'
};

directory.createAccount(picard, {passwordFormat: 'MCF'}, function (err, account) {
  // ..
});

File: source/code/ruby/account_management/create_account_mcf_hash.rb
account_data = {
  username: 'jlucpicard',
  email: 'captain@enterprise.com',
  given_name: 'Jean-Luc',
  surname: 'Picard',
  password: '$stormpath2$MD5$1$OGYyMmM5YzVlMDEwODEwZTg3MzM4ZTA2YjljZjMxYmE=$EuFAr2NTM83PrizVAYuOvw=='
}

account = directory.accounts.create(account_data, password_format: 'mcf')
